sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py

```diff
@@ -97,7 +96,8 @@ def register_schema(
         :param format: Format for the schema being registered.
          For now Avro is the only supported schema format by the service.
         :type format: Union[str, SchemaFormat]
-        :rtype: SchemaProperties
+        :rtype: ~azure.schemaregistry.SchemaProperties
+        :raises: :class:`~azure.core.exceptions.HttpResponseError:`
```

Comment:
nit: I don't think we need extra ":" at the end?
```suggestion
        :raises: :class:`~azure.core.exceptions.HttpResponseError`
```